PythonAccess/jedi.cpp

```diff
@@ -19,7 +19,7 @@ QStringList Jedi::AutoComplete(const QString& code, long row, long col) {
     PyGILState_STATE d_gstate;
     d_gstate = PyGILState_Ensure();
     // Build the name object
-    const char* pythonCode = code.toStdString().c_str();
```

Comment:
This was the bug. Somehow if you try invalid prefix then `pythonCode` will be pointing to some random address.

Reply:
Looks good can you add a comment for this.

Reply:
Also check if you type things like strings with Unicode letters (Hindi, Tamil, Chinese, etc..) that this works as well.

Reply:
Sure

Reply:
> Also check if you type things like strings with Unicode letters (Hindi, Tamil, Chinese, etc..) that this works as well.

You mean like this `someOtherLanguageText.`(click space + ctrl)

Reply:
Oh I see, it's not working. Understood encoding issue